src/block/block.jsx

```diff
@@ -23,7 +23,8 @@ export function block(blockName, mapPropsToModifiers = noop) {
                 const modifiers = mapPropsToModifiers(this.props);
                 return {
                     blockName,
-                    blockModifiers: classNames(modifiers)
+                    blockModifiers: classNames(modifiers),
+                    blockStyles: WrappedComponent.styles || {}
```

Comment:
If no styles specified new empty object is generated each time something changed